xml/decoders/esu/v3standardCVs.xml

```diff
@@ -189,13 +179,13 @@
         <decVal max="127"/>
         <label>Time between 2 chuffs @ speed step 1 (Range 0 - 127)</label>
         <label xml:lang="fr">délais d'accélaration entre deux crans (0 - 127)</label>
-        <label xml:lang="de">Geräuschmodus 1</label>
+        <label xml:lang="de">Abstand zweier Dampfstöße bei Fahrstufe 1</label>
       </variable>
       <variable CV="58" default="0" item="Sound Group 1 Option 1">
         <decVal max="127"/>
-        <label>Rate of decrease in time between 2 chuffs (Range 0 -127)</label>
-        <label xml:lang="fr">Délai de freinage entre deux crans(Range 0 -127)</label>
-        <label xml:lang="de">Geräuschmodus 2</label>
+        <label>Rate of decrease in time between 2 chuffs (Range 0-127)</label>
```

Comment:
There seems to be some inconsistency between ranges in this panel. Should they be ```START - STOP``` (note two spaces) or ```START-STOP``` (no spaces)?

Reply:
It should be START-STOP without spaces. I've edited the file with corrected ranges. Hope it works .